lib/Wasmtime.hs

```diff
@@ -904,7 +919,7 @@ newExternTypeFromPtr externtype_ptr = do
       (Ptr sub_ptr -> IO (Ptr sub_ptr)) ->
       (Ptr sub_ptr -> IO sub) ->
       IO sub
-    asSubType as_sub copy new = mask_ $ as_sub externtype_ptr >>= (copy >=> new)
+    asSubType as_sub cpy new = mask_ $ as_sub externtype_ptr >>= (cpy >=> new)
```

Comment:
brilliant

Reply:
I introduced the function `copy` below so I had to rename it here to prevent a warning.